core/includes/unicode.inc

```diff
@@ -110,15 +110,15 @@ function _unicode_check() {
 
   // Check for mbstring extension
   if (!function_exists('mb_strlen')) {
-    return array(UNICODE_SINGLEBYTE, $t('Operations on Unicode strings are emulated on a best-effort basis. Install the <a href="@url">PHP mbstring extension</a> for improved Unicode support.', array('@url' => 'http://www.php.net/mbstring')));
+    return array(UNICODE_SINGLEBYTE, $t('Operations on Unicode strings are emulated on a best-effort basis. Install the <a href="@url">PHP mbstring extension</a> for improved Unicode support.', array('@url' => 'https://www.php.net/mbstring')));
```

Comment:
@kiamlaluno if you run phpcs locally with B coding standards - isn't this line and the two following changes violations? The line is longer than 80 chars and in this case you should use line breaks in the array.

Reply:
I didn't touch issues reported from `phpcs` because there are many warnings and errors in the 34 changed files. There are more than 150 warnings and errors for a single file.

Reply:
Two more thingies, and we should be able to see, if phpcs / the action passes and the problem is just with annotations (so, the action), or something underneath is broken (phpcs / the rule(s)). That's important to figure out, where to report / fix.